python/adelphi/integration-tests/adelphi-docker-tests.sh

```diff
@@ -31,24 +29,41 @@ function create_schema() {
 	docker exec -it adelphi cqlsh -e "`cat base-schema.cql`" >> logs/$version.log
 }
 
-function anonymize() {
+function export_schema() {
 	local version=$1
-	echo "Anonymizing..."
-	adelphi export-cql > output/$version.cql 2>> logs/$version.log
+	echo "Exporting schemas for comparison..."
+	adelphi export-cql --no-metadata > output/$version-stdout.cql 2>> logs/$version.log
+	mkdir output/$version
+	adelphi --output-dir=output/$version export-cql 2>> logs/$version.log
 }
 
-function compare() {
-	local version=$1
-	echo "Comparing..."
-	diff schemas/$version.cql output/$version.cql
-	diffExitCode=$?
+function _do_diff() {
+        local schemaFile=$1
+	echo "Comparing schemas/${version}.cql to $schemaFile..."
+        diff -Z schemas/$version.cql $schemaFile
```

Comment:
I actually did see several cases where tests failed due to things like a missing newline at the end of one of the files fed into diff